cmd/clusterctl/pkg/client/init.go

```diff
@@ -23,6 +23,8 @@ import (
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/pkg/client/config"
 )
 
+const NopProvider = "-"
```

Comment:
Should this be `NoopProvider` as in No-Op, does nothing? 
```suggestion
const NoopProvider = "-"
```

Reply:
Go does have `ioutil.NopCloser`, fwiw